lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1492,7 +1493,7 @@ void SwiftASTContext::AddExtraClangArgs(std::vector<std::string> ExtraArgs) {
     if (!IsMacroDefinition(clang_argument))
       ApplyWorkingDir(clang_argument, cur_working_dir);
 
-    std::string clang_arg_str = clang_argument.str().str();
```

Comment:
@adrian-prantl not sure why this line was changed in `swift/master-next`, but I resolved the conflict to the variant that has been merged to `swift/master`

Reply:
`swift/master-rebranch` contains third variant of this code. I'll need some time to check if it builds at all.